docs.py

```diff
@@ -0,0 +1,89 @@
+import json
+import argparse
+
+from pkg_resources import get_distribution
+from pydantic_openapi_helper.core import get_openapi
+from pydantic_openapi_helper.inheritance import class_mapper
+from queenbee.recipe.recipe import Recipe
+from queenbee.operator.operator import Operator
+from queenbee.io.dag import DAGBooleanInput
+
+
+parser = argparse.ArgumentParser(description='Generate OpenAPI JSON schemas')
+
+parser.add_argument('--version',
+                    help='Set the version of the new OpenAPI Schema')
+
+args = parser.parse_args()
+
+VERSION = None
+
+if args.version:
+    VERSION = args.version.replace('v', '')
+else:
+    VERSION = '.'.join(get_distribution('dragonfly_schema').version.split('.')[:3])
+
+
+info = {
+    "description": "",
+    "version": VERSION,
+    "title": "",
+    "contact": {
+        "name": "Ladybug Tools",
+        "email": "info@ladybug.tools",
+        "url": "https://github.com/ladybug-tools/queenbee"
+    },
+    "x-logo": {
+        "url": "https://www.ladybug.tools/assets/img/honeybee.png",
+        "altText": "Queenbee logo"
+    },
+    "license": {
+        "name": "MIT",
+        "url": "https://github.com/ladybug-tools/queenbee-schema/blob/master/LICENSE"
+    }
+}
+
+
+modules = [
+    {'module': Recipe, 'name': 'Recipe'}
+    # {'module': Operator, 'name': 'Operator'}
```

Comment:
Should we not also generate Operator and Workflow Schemas?

Reply:
Good catch! Thanks.

Reply:
https://github.com/ladybug-tools/queenbee/issues/158